rook/README.md

```diff
@@ -77,9 +77,24 @@ $ kubectl create namespace kubestone
 
 ### `iperf`
 ```bash
-$ popper run -f workflows/iperf.yml
+# between server and client-1
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-one-hostname>
+
+# between server and client-2
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-two-hostname>
+
+# between server and client-3
+$ popper run -f workflows/iperf.yml -s _SERVER=<server-hostname> -s _CLIENT=<client-three-hostname>
+.
+.
 ```
 
+The hostnames to be provided to the substitution variables of the `popper run` command should be the `kubernetes.io/hostname` value of the nodes in the cluster.
+
+The workflow measures the link speed between a node allocated as a server and other nodes allocated as clients. An example plot from such a benchmark run is shown below.
```

Comment:
Based on previous commands, I thought it was the opposite, that is, speed from client to server (upload speed)?

Reply:
updated it. It is indeed from client to server